mava/components/updating/terminators.py

```diff
@@ -101,7 +102,7 @@ def on_parameter_server_run_loop_termination(
             and parameter_server.store.parameters[self.termination_key]
             > self.termination_value
         ):
-            print(
+            tf.print(
```

Comment:
Can we replace this with `logging.exception(` or `logging.error` . They seemed to print when I tested locally.

Reply:
Thanks @KaleabTessera :raised_hands: Great find.